filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -306,6 +306,9 @@ DECL_DRIVER_API_SYNCHRONOUS_N(backend::SyncStatus, getSyncStatus, backend::SyncH
 DECL_DRIVER_API_SYNCHRONOUS_N(bool, isWorkaroundNeeded, backend::Workaround, workaround)
 DECL_DRIVER_API_SYNCHRONOUS_N(void, setupExternalResource, intptr_t, externalResource)
 
+DECL_DRIVER_API_SYNCHRONOUS_0(bool, hasDepthResolveSupport)
+DECL_DRIVER_API_SYNCHRONOUS_0(bool, hasStencilResolveSupport)
```

Comment:
nit: `isDepthResolveSupported()` and `isStencilResolveSupported()` would be more in line with the naming of existing feature support query methods. (also extraneous empty line)

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29